src/tests/tracing/eventpipe/providervalidation/providervalidation.cs

```diff
@@ -35,20 +35,29 @@ public static int Main()
                 new EventPipeProvider("Microsoft-DotNETCore-SampleProfiler", EventLevel.Verbose)
             };
 
-            var ret = IpcTraceTest.RunAndValidateEventCounts(_expectedEventCounts, _eventGeneratingAction, providers, 1024);
+            bool enableRundown = TestLibrary.Utilities.IsNativeAot? false: true;
+
+            Dictionary<string, ExpectedEventCount> _expectedEventCounts = TestLibrary.Utilities.IsNativeAot? _expectedEventCountsNativeAOT: _expectedEventCountsCoreCLR;
+            var ret = IpcTraceTest.RunAndValidateEventCounts(_expectedEventCounts, _eventGeneratingAction, providers, 1024, enableRundownProvider:enableRundown);
             if (ret < 0)
                 return ret;
             else
                 return 100;
         }
 
-        private static Dictionary<string, ExpectedEventCount> _expectedEventCounts = new Dictionary<string, ExpectedEventCount>()
+        private static Dictionary<string, ExpectedEventCount> _expectedEventCountsCoreCLR = new Dictionary<string, ExpectedEventCount>()
         {
             { "MyEventSource", new ExpectedEventCount(100_000, 0.30f) },
             { "Microsoft-Windows-DotNETRuntimeRundown", -1 },
             { "Microsoft-DotNETCore-SampleProfiler", -1 }
         };
 
+        private static Dictionary<string, ExpectedEventCount> _expectedEventCountsNativeAOT = new Dictionary<string, ExpectedEventCount>()
+        {
+            { "MyEventSource", 100_000 },
```

Comment:
Does coreclr need the `ExpectedEventCount` error, but nativeaot does not?

Reply:
I believe `coreclr ` error range tolerance is an artifact of earlier times when `EventPipe`was dropping events. NativeAOT doesn't support rundown and `sampleprofiler `and followed a policy of not touching `coreclr` testing.